sdk/daml-assistant/get-daml.sh

```diff
@@ -149,7 +149,7 @@ elif [ "$major" -ge "3" ]; then
       platform=-x86_64
       ;;
     linux-aarch64)
-      platform=-aarch64
```

Comment:
I don't think you want this change, as it looks like for 3 and up they do have these platform-specific artifacts, e.g.

https://github.com/digital-asset/daml/releases/tag/v3.2.0-snapshot.20240911.0

Reply:
Agree, we do have these artifacts for 3x, so this part should be undone.